ORK1Kit/ORK1Kit/Common/ORK1AnswerFormat.m

```diff
@@ -1039,7 +1041,8 @@ - (BOOL)isEqual:(id)object {
     return (ORK1EqualObjects(self.text, castObject.text)
             && ORK1EqualObjects(self.detailText, castObject.detailText)
             && ORK1EqualObjects(self.value, castObject.value)
-            && self.exclusive == castObject.exclusive);
+            && self.exclusive == castObject.exclusive
+            && self.detailTextShouldDisplay == castObject.detailTextShouldDisplay);
```

Comment:
I'm not sure what the standard behavior is in ResearchKit: since `detailTextShouldDisplay` can change at runtime based on UI interactions, should it be included in isEqual and encode/decode operations?

Reply:
Good point - adjusted